lib/pure/mimetypes.nim

```diff
@@ -1917,6 +1917,33 @@ func register*(mimedb: var MimeDB, ext: string, mimetype: string) =
   {.noSideEffect.}:
     mimedb.mimes[ext.toLowerAscii()] = mimetype.toLowerAscii()
 
+
+since (1, 5):
+  func mimesLongest(): array[2, int] {.compiletime.} =
+    runnableExamples:
+      static:
+        doAssert mimesLongest() >= (ext: 24, mime: 73)
```

Comment:
@juancarlospaco this is bad for several reasons:
* runnableExamples inside a private proc is never run/compiled
* the code is in fact invalid (>= not defined on array)
* even if it were valid (say it returned a tuple), it's bad because  `(23, 73) >= (22, 100)`


and as reported elsewhere it causes this regression: https://github.com/nim-lang/Nim/issues/16703